test/parallel/test-release-changelog.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+// This test checks that the changelogs contain an entry for releases.
+
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+
+const getDefine = (text, name) => {
+  const regexp = new RegExp(`#define\\s+${name}\\s+(.*)`);
+  const match = regexp.exec(text);
+  assert.notStrictEqual(match, null);
+  return match[1];
```

Comment:
```suggestion
  return match?.[1];
```
Otherwise it's going to throw for non-release commits, instead I think we want to skip.

Reply:
If the assert fails it means [`node_version.h`](https://github.com/nodejs/node/blob/v18.x/src/node_version.h) is malformed. Even for non-release commits the #define's in that file have values.

For non-release commits the test does skip https://github.com/nodejs/node/blob/3267e34e52293083d66c55d43843e7d53c4861a9/test/parallel/test-release-changelog.js#L23-L25